### PR TITLE
Revert "ergoCubV1_1: remove ft sensors of the arms"

### PR DESCRIPTION
**Pull request by Nicogene**
This reverts commit 6be5893296bdf465323ab3ca698745d7b74dc3a3.

cc @GiulioRomualdi

### CODE REVIEW

File: tests/ergocub-model-test.cpp [hunk 1]
```diff
@@ -449,7 +449,7 @@ bool checkFTSensorsAreOddAndNotNull(iDynTree::ModelLoader & mdlLoader)
 }
 
 /**
- * All the ergoCub have a even and not null number of F/T sensors.
+ * All the iCub have a even and not null number of F/T sensors.
```
Comment (traversaro) [inline tests/ergocub-model-test.cpp:452]: Is this change intentional?

**Review by traversaro — APPROVED** [review]

File: tests/ergocub-model-test.cpp (hunk 1 above)
Comment (Nicogene) [reply tests/ergocub-model-test.cpp:452]: No I fixed in the previous commit and reverting it put back iCub
